subhub/tests/unit/test_payment_calls.py

```diff
@@ -525,11 +527,40 @@ def test_update_payment_method_invalid_stripe_customer(
     assert "No such customer:" in exception.user_message
 
 
-def test_customer_update_success(create_subscription_for_processing):
-    (subscription, code) = create_subscription_for_processing
-    (data, code) = payments.customer_update("process_test")
+def test_customer_update_success(monkeypatch):
+
+    subhub_account = MagicMock()
+
+    get_user = MagicMock()
+    user_id = PropertyMock(return_value="user123")
+    cust_id = PropertyMock(return_value="cust123")
+    type(get_user).user_id = user_id
+    type(get_user).cust_id = cust_id
+
+    subhub_account.get_user = get_user
+
+    stripe_customer = Mock(
+        return_value={
+            "metadata": {"userid": "user123"},
+            "subscriptions": {"data": []},
+            "sources": {
+                "data": [
+                    {
+                        "funding": "blah",
+                        "last4": "1234",
+                        "exp_month": "02",
+                        "exp_year": "2020",
+                    }
+                ]
+            },
+        }
+    )
+
+    monkeypatch.setattr("flask.g.subhub_account", subhub_account)
+    monkeypatch.setattr("stripe.Customer.retrieve", stripe_customer)
+
+    (data, code) = payments.customer_update("user123")
```

Comment:
we just removed all of the `(x, y) = some_func()` expressions; let's be consistent; don't need the parens around the multiple items returned